OpenEdXMobile/src/main/java/org/edx/mobile/view/EditUserProfileFragment.java

```diff
@@ -109,6 +106,83 @@ public class EditUserProfileFragment extends BaseFragment implements BaseFragmen
     @NonNull
     private final ImageCaptureHelper helper = new ImageCaptureHelper();
 
+    private final ActivityResultLauncher<Intent> photoCropLauncher =
+            registerForActivityResult(new ActivityResultContracts.StartActivityForResult(), result -> {
+                Intent resultData = result.getData();
+                if (result.getResultCode() == Activity.RESULT_OK && resultData != null) {
+                    final Uri imageUri = CropImageActivity.getImageUriFromResult(resultData);
+                    final Rect cropRect = CropImageActivity.getCropRectFromResult(resultData);
+                    if (imageUri != null && cropRect != null) {
+                        final Task<Void> task = new SetAccountImageTask(requireActivity(), username, imageUri, cropRect);
+                        task.setProgressDialog(viewHolder.profileImageProgress);
+                        executePhotoTask(task);
+                        analyticsRegistry.trackProfilePhotoSet(CropImageActivity.isResultFromCamera(resultData));
+                    }
+                }
+            });
+
+    private final ActivityResultLauncher<Intent> photoChooserLauncher =
+            registerForActivityResult(new ActivityResultContracts.StartActivityForResult(), result -> {
+                Intent resultData = result.getData();
+                if (result.getResultCode() == Activity.RESULT_OK && resultData != null) {
+                    Uri imageUri = resultData.getData();
+                    if (imageUri != null) {
+                        photoCropLauncher.launch(
```

Comment:
either move this into one line or change others accordingly.

Reply:
Moved to one line.